packages/jest-circus/src/__mocks__/testEventHandler.ts

```diff
@@ -7,7 +7,10 @@
 
 import {Circus} from '@jest/types';
 
-const testEventHandler: Circus.EventHandler = (event, state) => {
+const testEventHandler: Circus.EventHandler = (
+  event: Circus.Event,
```

Comment:
are you sure we need these explicit parameter types when the whole variable is typed?

Reply:
I am almost sure, yes.
Last time I checked, it kept throwing TypeScript errors if I write it any other way.
I'll recheck soon with the latest stable TypeScript and post a screenshot of the error.

Reply:
Surprisingly, it works now with the latest TypeScript. Thanks!